### PR TITLE
Add unstyled vertical tabs

### PR DESCRIPTION
**Pull request by Magnus-G**
image

### CODE REVIEW

File: source/stylesheets/refills/unstyled/_vertical-tabs.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      width: 20%;
+    }
+  }
+
+  .vertical-tab {
+    @include media($large-screen) {
+      display: block;
+
+      &.is-active {
+        color: #ff0000;
```
Comment (Magnus-G) [inline source/stylesheets/refills/unstyled/_vertical-tabs.scss]: This guideline... I don't know.

Comment (tysongach) [inline source/stylesheets/refills/unstyled/_vertical-tabs.scss]: He’s wrong. Something got added to the linter that shouldn't have and needs to be fixed.

File: source/stylesheets/refills/unstyled/_vertical-tabs.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
+.vertical-tabs-container {
+  $large-screen: em(640) !default;
+  $vertical-tabs-height: em(100);
```
Comment (tysongach) [inline source/stylesheets/refills/unstyled/_vertical-tabs.scss]: Put variables outside of declaration.

Comment (kaishin) [inline source/stylesheets/refills/unstyled/_vertical-tabs.scss]: I think variables that are used for one component should remain within its scope. We gain nothing by putting them outside, beside potential conflicts with other components.

**Pull request closed without merging**